### ISSUE TITLE
BUG: on bad input error dialogs should appear

### ISSUE THREAD

**Issue by ElSkunkito** [VertNet/georefcalculator#49]
Repro
   edit test_data.js, g_tests[5]
   change
   FROM TextFieldOffset: "3",
   TO  TextFieldOffset: "3 mi",
   save 
run '5 test ' using the selectorizor at the top

Actual Results:
   No errors alerted(other than console, which the end user wont see)
   note the NaN for Decilmal Latictude in results

Expected:Results:
    Error Dialog
    TextFiledOffset get set to 0
    results get cleared

Basically this mean renable the try catch logic thats been commented out.

**Comment by ElSkunkito** [comment]
I'm ready to call this closed with commit 6ec6a97..42b78d0
